src/Appwrite/Specification/Format/Swagger2.php

```diff
@@ -401,11 +401,20 @@ public function parse(): array
 
         foreach ($this->models as $model) {
             foreach ($model->getRules() as $rule) {
-                if (
-                    in_array($model->getType(), $usedModels)
-                    && !in_array($rule['type'], ['string', 'integer', 'boolean', 'json', 'float'])
-                ) {
-                    $usedModels[] = $rule['type'];
+                if (!in_array($model->getType(), $usedModels)) {
```

Comment:
Why this change came in this PR?

Reply:
We now introduced new (yet unsupported) concept to Swagger.
It's a definition that is only referenced in a definition, not in a path.

These changes make it work, screenshots in this PR: https://github.com/appwrite/docs/pull/202